test/base-01-errors-startup.js

```diff
@@ -0,0 +1,28 @@
+var tape = require('tape');
+var mjAPI = require('../lib/main.js');
+
+// NOTE this test must first (in tape) to test the startup phase (hence the filename contains "01")
+
+tape('Catch errors during startup phase', function(t) {
+  t.plan(1);
+  mjAPI.config({
+    extensions: 'blargh'
+  });
+  mjAPI.typeset(
+    {
+      math: 'x',
+      format: 'TeX',
+      mml: true
+    },
+    function(data) {
+      t.ok(
+        data.errors,
+        'Error (loading non-existent extension) is caught in output'
+      );
+     // reset configuration
+     mjAPI.config({
+        extensions: 'tex2jax'
```

Comment:
I don't think this is a reset, actually. The `text2jax` extension is not loaded by mathjax-node, so this is an alteration.  I think you want

```
extensions: ''
```

since the default extensions string is empty.

Reply:
Copy that.